README.md

```diff
@@ -97,6 +97,10 @@ environment:
   - RSYSLOG_REMOTE_TEMPLATE=RSYSLOG_ForwardFormat
 ```
 
+#### Advanced logging configuration
+
+If configuration via environment variables is not flexible enough it's possible to configure rsyslog directly: `.conf` files in the `/etc/rsyslog.d` will be [sorted alphabetically](https://www.rsyslog.com/doc/v8-stable/rainerscript/include.html#file) and included into the primary configuration.
+
```

Comment:
maybe "...the `/etc/rsyslog.d` _directory_ will be ..."

Reply:
It was there, has been lost during multiple editing iterations. Thanks, fix comes in a second.